app/application.py

```diff
@@ -671,9 +561,15 @@ def register_enter_your_details():
 
         # Step 2
         # Register with the party service
-        registrationData = {'emailAddress': email_address, 'firstName': first_name, 'lastName': last_name, 'telephone': phone_number, 'status': 'CREATED'}
+        registrationData = {'emailAddress': email_address,
+                            'firstName': first_name,
+                            'lastName': last_name,
+                            'telephone': phone_number,
+                            'status': 'CREATED'}
+
         headers = {'authorization': encoded_jwt_token, 'content-type': 'application/json'}
-        partyServiceURL = PartyService.PARTYSERVICE_PROTOCOL + PartyService.PARTYSERVICE_SERVER + PartyService.PARTYSERVICE_RESPONDENTS_ENDPOINT
+
+        partyServiceURL = Config.API_GATEWAY_PARTY_URL + 'respondents'
```

Comment:
This is much more clear which is great! Good job. ;-) ...
But if you make the creation of the URL with things like protocol, server and endpoints it means you can change things more granularly in the future. Which might be good when you have multiple endpoints e.g. the OAuth2 server now has 4 endpoints and can support http or https if we put an TLS layer in front of the server.
With great power comes great responsibility !!! If there are lots of changes required to party service URL you can maintain this! :-)